packages/components/CHANGELOG-FIGMA-COMPONENTS.md

```diff
@@ -1,4 +1,23 @@
 # [HDS Product - Components](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?type=design&node-id=6790-10926&mode=design&t=Ps0aMGZ6F3z7bAJ4-0)
+## December 20th, 2024
+
+`IconTile` - Visual updates made to the component, including removing the border and updating colors for improved contrast and visual alignment.
+
+`[Figma Library v1.0](https://www.figma.com/design/noyY6dUMDYjmySpHcMjhkN/HDS-Components?m=auto&node-id=2-7&t=CPG3tlUvMHeQdEos-1)` - **Deprecated**. It will no longer receive updates or support. Instead, use the `[Figma Library v2.0](https://www.figma.com/design/iweq3r2Pi8xiJfD9e6lOhF/HDS-Components-v2.0?m=auto&node-id=2-7&t=9BNZJMDOrABsgaJQ-1)`
+
+`Figma Library 2.0` - Released. Figma Library link
+
+`Foundations Library 1.0` - **Deprecated**. It will no longer receive updates or support.
+
+`Foundations Library 2.0` - Released.
+
+`Product Patterns 1.0` - **Deprecated**. It will no longer receive updates or support.
+
+`Product Patterns 2.0` - Released. Figma Library link
```

Comment:
1. Let's use the official name of the files.
2. Components and Foundations have separate sections in the release notes. Foundation updates should be moved to the Foundations section.
3. We'll need to add a new Patterns section.
4. There was no Patterns 1.0 to deprecate.
5. Let's keep the same formatting throughout.
6. Figma library link is above, so don't think it needs to be duplicated here.

```suggestion
`HDS Components UI Kit v1.0` - **Deprecated** the first version of our components UI kit. It will no longer receive updates or support. 

`HDS Components UI Kit v2.0` - Released a new version of our components UI kit.
```